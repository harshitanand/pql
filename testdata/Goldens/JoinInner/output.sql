WITH "subquery0" AS (SELECT "State" AS "State" FROM "StormEvents"),
"subquery1" AS (SELECT upper("State") AS "State", "StateCapital" AS "StateCapital" FROM "StateCapitals")
SELECT * FROM "subquery0" AS "$left" JOIN "subquery1" AS "$right" ON "$left"."State" = "$right"."State" ORDER BY "State" ASC NULLS FIRST;
