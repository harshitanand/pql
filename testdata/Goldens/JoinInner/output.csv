FLORIDA,FLORIDA,Tallahassee
FLORIDA,FLORIDA,Tallahassee
GEORGIA,GEORGIA,Atlanta
MISSISSIPPI,MISSISSIPPI,Jackson
